app/uk/gov/hmrc/cataloguefrontend/DependencyExplorerController.scala

```diff
@@ -135,12 +135,12 @@ class DependencyExplorerController @Inject()(
                                       service
                                         .getServicesWithDependency(team, flag, query.group, query.artefact, versionRange, scope)
                                     }
-                    pieData      =  PieData(
+                    pieData      =  if (results.nonEmpty) Some(PieData(
                                         "Version spread"
                                       , results
                                           .groupBy(r => s"${r.depGroup}:${r.depArtefact}:${r.depVersion}")
                                           .map(r => r._1 -> r._2.size)
-                                      )
+                                      )) else None
```

Comment:
If its Some, but with empty results, the PieChart library from Google errors with TypeErrors